cmd/cerbosctl/get/policy.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021-2022 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package get
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	policyv1 "github.com/cerbos/cerbos/api/genpb/cerbos/policy/v1"
+	"github.com/cerbos/cerbos/client"
+	"github.com/cerbos/cerbos/cmd/cerbosctl/internal"
+	"github.com/cerbos/cerbos/internal/policy"
+)
+
+func listPolicies(c client.AdminClient, cmd *cobra.Command, args *Arguments, resType ResourceType) error {
+	policyIds, err := c.ListPolicies(context.Background())
+	if err != nil {
+		return fmt.Errorf("error while requesting policies: %w", err)
+	}
+
+	if !args.NoHeaders {
+		err = internal.PrintPolicyHeader(cmd.OutOrStdout())
+		if err != nil {
+			return fmt.Errorf("failed to print hedaer: %w", err)
+		}
+	}
+
+	for idx := range policyIds {
+		if idx%internal.MaxIDPerReq == 0 {
+			idxEnd := internal.MinInt(idx+internal.MaxIDPerReq, len(policyIds))
+			policies, err := c.GetPolicy(context.Background(), policyIds[idx:idxEnd]...)
+			if err != nil {
+				return fmt.Errorf("error while requesting policy: %w", err)
+			}
+
+			filtered := filterPolicies(policies, policyIds[idx:idxEnd], getArgs.Name, getArgs.Version, resType)
+
+			err = internal.PrintIds(cmd.OutOrStdout(), filtered...)
```

Comment:
since we have already retrieved the policies, why not also print out the name and version in addition to the ID